src/test/resources/__files/body-heads-master-4KaYG.json

```diff
@@ -0,0 +1 @@
+{"ref":"refs/heads/master","url":"https://api.github.com/repos/testOwner/testRepo/git/refs/heads/master","object":{"sha":"815cd56c820c53c665d78f409749a62ad5ec9a48","type":"commit","url":"https://api.github.com/repos/testOwner/testRepo/git/commits/815cd56c820c53c665d78f409749a62ad5ec9a48"}}
```

Comment:
I agree; that's the format it comes back in from the wiremock recorder.  Since this is something that will have to be uploaded manually, I can change this to be formatted.